people.md

```diff
@@ -22,50 +22,40 @@ Dr. Ben van Werkhoven
 ---------------------
 Netherlands eScience Center
 
-
 Domain Experts
 ==============
 
-Jerom Aerts, MSc.
------------------
+Jerom Aerts, MSc
+----------------
 Delft​ ​ University​ ​ of​ ​ Technology
 
 Engineers
 =========
 
-Dr. Berend Weel
```

Comment:
It would be nice to list engineers that have worked on the project in the past. To not have them listed seems unfair to me.

Could we add them their own section at the bottom of the page?